lib/sidekiq_unique_jobs/middleware/server.rb

```diff
@@ -6,6 +6,8 @@ module Middleware
     #
     # @author Mikael Henriksson <mikael@mhenrixon.com>
     class Server
+      include Sidekiq::ClientMiddleware if defined?(Sidekiq::ServerMiddleware)
```

Comment:
@mhenrixon Shouldn't this include ServerMiddleware?

Reply:
Fixed it now